### DISCUSSION TITLE
MapMagic2 Integration for Nature Renderer 2022

### DISCUSSION THREAD

**Discussion by Logophoman** [visualdesigncafe/nature-renderer#36] [Help]
Previously, Nature Renderer had MapMagic 2 support out of the box... [See the docs...](https://v3.visualdesigncafe.com/nature-renderer/docs/2020/map-magic) which worked back then. 

However it seems to me that following the instructions provided for the 2020 version won't work with the 2022 version and the recent MapMagic2 version anymore. 

I am using  Unity 2022.3.22f1 at the time... 

Any Ideas? Did anyone figure that out or is there something I'm missing? 

![image](https://github.com/visualdesigncafe/nature-renderer/assets/28789967/4ee736fc-b894-48c2-92e6-ef0971ecf42c)

**Comment by mroshaw** [comment]
I haven't used MM, I'm afraid, but the error you're seeing is because the Nature Renderer component is on a game object that doesn't have a Terrain component - that's fairly fundamental for NR to work.

If MM creates a standard Unity terrain, then find the game object with the Terrain component, and add NR there. Worth a try, though maybe the dev will add official support to MM in the future.

**Comment by maxplooi** [comment]
It looks like you are using an older version of Nature Renderer, this issue has been fixed since version [2022.0.1](https://visualdesigncafe.com/nature-renderer/docs/2022/release-notes#202201). You can go to **Help > Nature Renderer > Check for Updates** in the Unity editor to download the latest version.

**Reply by Logophoman** [reply]
Thanks so much! I'll take a look later!
